pyproject.toml

```diff
@@ -41,4 +41,12 @@ sc4snmp-traps = "splunk_connect_for_snmp_traps.snmp_trap_server:main"
 requires = ["poetry>=0.12"]
 build-backend = "poetry.masonry.api"
 
+[tool.isort]
+profile = "black"
 
+[[tool.mypy.overrides]]
```

Comment:
If we are using `pre-commit`, this section is not needed due to `additional_dependencies: [types-all]` in `.pre-commit-config.yaml` file.

It works fine without these lines, I checked locally.

Reply:
Deleted it